controller/space_test.go

```diff
@@ -144,6 +151,12 @@ func (rest *TestSpaceREST) TestSuccessCreateSpaceAndDefaultArea() {
 	assert.Len(t, areaList.Data, 1)
 	assert.Equal(t, name, *areaList.Data[0].Attributes.Name)
 
+	// verify if default iteration is created or not
+	spaceIterationSvc, spaceIterationCtrl := rest.SecuredSpaceIterationController(testsupport.TestIdentity)
+	_, iterationList := test.ListSpaceIterationsOK(t, spaceIterationSvc.Context, spaceIterationSvc, spaceIterationCtrl, createdID)
+	assert.Len(t, iterationList.Data, 1)
```

Comment:
done in https://github.com/almighty/almighty-core/pull/990/commits/437523515bf06e594b2974d864f367e60a728b39#diff-0996e94a9c00b464ed5e411c5beedf30R157